akka-stream/src/main/scala/akka/stream/stage/GraphStage.scala

```diff
@@ -55,7 +58,7 @@ abstract class GraphStage[S <: Shape] extends Graph[S, Unit] {
  *    of the enclosing [[GraphStage]]
  *  * Possible mutable state, accessible from the [[InHandler]] and [[OutHandler]] callbacks, but not from anywhere
  *    else (as such access would not be thread-safe)
- *  * The lifecycle hooks [[preStart()]] and [[postStop()]]
+ *  * The lifecycle hooks preStart() and postStop()
```

Comment:
yes, most of them work just fine, there are special cases that will not work, but those are rare. Testing with java8 javadoc is not a priority right now, so it's better to have working scaladoc links now, and we can fix the broken once later instead of not having any links at all.